src/main/java/com/algorand/algosdk/account/Account.java

```diff
@@ -273,6 +274,29 @@ public Signature signBytes(byte[] bytes) throws NoSuchAlgorithmException {
         }
     }
 
+    /**
+     * Sign the given bytes, and wrap in Signature.
+     * @param bytes the data to sign
+     * @return a signature
```

Comment:
for consistency (in the java interface), I'd prefer we return a Signature object (and the client can call sig.getBytes). Is there a strong argument for returning raw bytes?